.github/workflows/Tests.yaml

```diff
@@ -531,13 +507,10 @@ jobs:
           -D CHARM_ROOT=/work/charm_6_10_2/multicore-linux-x86_64-${CHARM_CC}
           -D CMAKE_BUILD_TYPE=Debug
           -D DEBUG_SYMBOLS=OFF
-          -D RUN_TESTS_IN_TEST_EXECUTABLES=OFF
           -D STRIP_SYMBOLS=ON
           -D STUB_EXECUTABLE_OBJECT_FILES=ON
```

Comment:
Also delete these lines (I think they just save disk space, but you're compiling only a few files)

Reply:
So why remove them?

Reply:
🤷‍♂️ seems overkill for those few compiled libs.. but can't hurt I guess